### PR TITLE
gomplate 3.8.0

### PR DESCRIPTION
**Pull request by hairyhenderson**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
==> make compress VERSION=3.8.0
Makefile:63: *** Wrong upx version - need 3.94.  Stop.
```

**Comment by hairyhenderson** [comment]
@chenrui333 Thanks for the heads-up! Getting this running with 3.94 is going to be frustrating, I'm guessing... The v3.7.0 binary that I shipped using UPX 3.96 (current) had a bunch of random intermittent failures reported by end-users, so I reverted to 3.94 (3.95 was pretty well busted for a while too).

I'm thinking the best way to fix this in the short (and possibly long) term is just to drop UPX entirely. I'll update this PR to build without it.

File: Formula/gomplate.rb [hunk 1]
```diff
@@ -14,11 +14,13 @@ class Gomplate < Formula
...
 
   depends_on "go" => :build
-  depends_on "upx" => :build
+  # Temporarily disabled upx
 
   def install
-    system "make", "compress", "VERSION=#{version}"
-    bin.install "bin/gomplate-slim" => "gomplate"
+    # system "make", "compress", "VERSION=#{version}"
+    # bin.install "bin/gomplate-slim" => "gomplate"
```
Comment (SMillerDev) [inline Formula/gomplate.rb:21]: No need to keep the commented code.

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
Thanks @hairyhenderson!!

**Pull request closed without merging**